docs/rules/prefer-screen-queries.md

```diff
@@ -2,8 +2,8 @@
 
 ## Rule Details
 
-DOM Testing Library (and other Testing Library frameworks built on top of it) exports a `screen` object which has every query (and a `debug` method). This works better with autocomplete and makes each test a little simpler to write and maintain.
-This rule aims to force writing tests using queries directly from `screen` object rather than destructuring them from `render` result.
+DOM Testing Library (and other Testing Library frameworks built on top of it) exports a `screen` object which has every query (and a `debug` method). This works better with autocomplete and makes each test a little simpler to write and maintain.  
+This rule aims to force writing tests using queries directly from `screen` object rather than destructuring them from `render` result. Given the screen component does not expose utility methods such as `rerender()` or the `container` property, it is correct to use the `render` response in those scenarios-
```

Comment:
There is a weird dash at the end of the sentence. Could you fix it please?

Reply:
uops, typo

Reply:
fixed